docs/docs/exporting.md

```diff
@@ -18,8 +18,8 @@ only support `.yaml` files.
 For that case, `tk export` can be used:
 
 ```bash
-#           <environment>         <outputDir>
-$ tk export environments/promtail promtail
+#           <outputDir> <environment>
```

Comment:
Maybe add a note that this changed recently?

Reply:
:+1: in #495